src/insns/sh4add_32bit.adoc

```diff
@@ -35,6 +35,9 @@ Increment the address field of `cs2` by `rs1` shifted left by 4 bit positions. C
 Legacy Mode Description::
 Increment `rs2` by `rs1` shifted left by 4 bit positions.
 
+Exceptions::
+include::require_cre.adoc[]
```

Comment:
Do we really need CRE for this instruction? I would assume it works fine in legacy mode?